docs/howto/how-to-make-api-tool/api-tool-guide.md

```diff
@@ -0,0 +1,250 @@
+# Creating a tool with an API hit
+In this guide, we will use the [Alteryx Python SDK](https://pypi.org/project/ayx-python-sdk/) and [Alteryx Plugin CLI](https://pypi.org/project/ayx-plugin-cli/) to create a tool that pulls information from an API and find the mean, min, and max of the data.
+
+---
+## Workspace Setup
+---
+### 1. Creating a workspace
+The very first step to creating a plugin is to make a plugin workspace. We initialize a plugin workspace in an empty directory with the `sdk-workspace-init` command. Run the command and fill out the prompts, which will then start the workspace initialization process.
+
+```
+$ ayx_plugin_cli sdk-workspace-init
+Package Name: API Tool
+Tool Category [Python SDK Examples]: 
+Description []: API Tool
+Author []: Alteryx
+Company []: Alteryx
+Backend Language (python): python
+[Workspace initialization] started
+[Workspace initialization] .  Create configuration directory
+[Workspace initialization] .  Create DCM Schemas directory
+[Workspace initialization] .  Create .gitignore
+[Workspace initialization] .  Create README.md
+[Workspace initialization] .  Initialize backend
+[Workspace initialization] Creating ~\sdk-api-tool\backend\ayx_plugins
+[Workspace initialization] Creating file ~\sdk-api-tool\backend\requirements-local.txt
+[Workspace initialization] Creating file ~\sdk-api-tool\backend\requirements-thirdparty.txt
+[Workspace initialization] Creating file ~\sdk-api-tool\backend\setup.py
+[Workspace initialization] Creating file ~\sdk-api-tool\backend\ayx_plugins\__init__.py
+[Workspace initialization] .  Create tests directory
+[Workspace initialization] .  Initialize UI
+[Workspace initialization] finished
+Created Alteryx workspace in directory: ~\sdk-api-tool
+Workspace settings can be modified in: ayx_workspace.json
+[Generating config files] started
+[Generating config files] .  generate_config_files:generate_config_xml
+[Generating config files] Generating top level config XML file...
+[Generating config files] finished
+```
+
+### 2. Creating a plugin
+The next step is to add a plugin to the workspace. We do this using the `create-ayx-plugin` command. Fill out the prompts and then you will have the template code for your SDK Plugin. For this tool, we are choosing the `Input` tool type.
```

Comment:
TW review - update to:

The next step is to add a plugin to the workspace. To do so, use the `create-ayx-plugin` command. Reply to the prompts and then you will have the template code for your SDK Plugin. For this tool, we use the `Input` tool type.